doc/source/cluster/vms/user-guides/logging.md

```diff
@@ -0,0 +1,32 @@
+# Log Persistence
+
+Logs (both system and application logs) are useful for troubleshooting Ray applications and system. For example, you may want to access system logs if a node dies unexpectedly.
+
+Similar to Kubenetes, Ray does not provide a native storage solution for log data. Users need to manage the lifecycle of the logs by themselves. The following sections provide some instructions on how to collect logs from Ray clusters running on VMs.
+
+## Ray log directory
+By default, Ray writes logs to files in the directory `/tmp/ray/session_*/logs` on each Ray node's file system, including application logs and system logs. Learn more about the {ref}`log directory and log files <logging-directory>` before you start to collect the logs.
+
+
+## Log processing tools
+
+There are a number of open source log processing tools available, e.g, [Vector][Vector] and [Fluent Bit][FluentBit].
+Other popular tools include [Fluentd][Fluentd], [Filebeat][Filebeat], and [Promtail][Promtail].
```

Comment:
```suggestion

```

Reply:
Fixed the issue. We shouldn't remove this line because the list below is just to manage the links in a central place and is not visible in the doc itself.